BotRoyaleUnity/Assets/Scripts/Networking/SocketConnectionHandler.cs

```diff
@@ -126,10 +126,40 @@ public void EmitGameMessage(JSONObject data, Action<string> onMessageSent = null
         }
         else
         {
-            socket.Emit("game-message", data.ToString(), onMessageSent); ;
+            socket.Emit("game-message", data.ToString(), onMessageSent);
         }
     }
 
+    public void EmitCurrentParts(GameObject robot){
```

Comment:
I think this method should be in a different class. This class is a general network connection API, this stuff that's very specific to the robots would make more sense to be in one of the robot classes